active/0001-ember-doctor.md

```diff
@@ -0,0 +1,49 @@
+- Start Date: 2015-01-10
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+We need a way to run diagnostics on Ember CLI based projects to let developers know about potential system level incompatibilities. Developers should also be able to get a bill of health for their project for things like outdated dependencies.  This bill of health should also be extensible.
+
+# Motivation
+
+The motivation behind this is 2 pronged:
+
+1. Allows developers to submit system level information in pull requests, so that bugs can be filed and potentially replicated.
+2. Gives developers the ability to know about the health of their project and to potentially help with stagnation.
+
+# Detailed design
+
+The design for this is rather simple. We would first introduce a command called `ember doctor` that would run some default checks. The default checks would do the following:
+
+- Run `ember v --verbose` and complain loudly for incompatible versions
+- Run `npm outdated --depth 0` to check on outdated modules
+- Run `bower list` and display out of date bower components
```

Comment:
I was not aware of ember-cli-dependecy-checker till yesterday, I agree it should get ran.  :+1: to OS level stuff.